nss-day-cohort-43/nuttyshell-sons-of-witches
# Bs working deletion functionality

# Description
Add functionality to prepend an @ symbol at the front of the username when a new message is created and to populate an Add Friend button when the user clicks the username associated with a message someone else submitted.

Fixes # (issue)
## Type of change
Please delete options that are not relevant.

# Testing Instructions
Please ensure an @ symbol appears at the beginning of the usernames of submitted messages and that clicking the username associated with submitted messages populates an Add Friend button

# Checklist:
